flytekit/common/component_nodes.py

```diff
@@ -67,7 +67,14 @@ def __init__(self, sdk_workflow=None, sdk_launch_plan=None):
         """
         self._sdk_workflow = sdk_workflow
         self._sdk_launch_plan = sdk_launch_plan
-        super(SdkWorkflowNode, self).__init__()
```

Comment:
You shouldn't remove this in case we ever add `__init__` logic in the super

Reply:
https://github.com/lyft/flytekit/pull/92/commits/a3a389627a6cda42fc9a0b86b33c0dec7f7319c7#diff-11888e82c8c56a7d42b7bc092340d1baR70?